### PR TITLE
Update fnc_vehicleAction.sqf

### PR DESCRIPTION
**Pull request by nomisum**
fix plow abuse

### CODE REVIEW

File: addons/functions/functions/fnc_vehicleAction.sqf [hunk 1]
```diff
@@ -133,10 +133,10 @@ if (hasInterface) then {
 		true,
 		true,
 		"",
-		"((_target getVariable ['grad_trenches_functions_plowlowered', 0]) == 1)", // _target, _this, _originalTarget
+		"((_target getVariable ['grad_trenches_functions_plowlowered', 0]) == 1) && player == driver _target", // _target, _this, _originalTarget
```
Comment (Fusselwurm) [inline addons/functions/functions/fnc_vehicleAction.sqf:136]: do use `ACE_player` or `call CBA_fnc_currentUnit` so feature works for Zeus while using remote control

Comment (Fusselwurm) [reply addons/functions/functions/fnc_vehicleAction.sqf:136]: oooh or use `_this` , apparently it's the caller / the unit the action is shown to

https://community.bistudio.com/wiki/addAction

File: addons/functions/functions/fnc_vehicleAction.sqf [hunk 2]
```diff
@@ -133,10 +133,10 @@ if (hasInterface) then {
...
 		50,
```
Comment (Fusselwurm) [inline addons/functions/functions/fnc_vehicleAction.sqf:137]: reduce radius  to 10m or so (which is still very generous)

File: addons/functions/functions/fnc_vehicleAction.sqf (hunk 1 above)
Comment (AdlerSalbei) [reply addons/functions/functions/fnc_vehicleAction.sqf:136]:
```suggestion
		"((_target getVariable ['grad_trenches_functions_plowlowered', 0]) == 1) && (call CBA_fnc_currentUnit) == driver _target", // _target, _this, _originalTarget
```

File: addons/functions/functions/fnc_vehicleAction.sqf (hunk 2 above)
Comment (AdlerSalbei) [reply addons/functions/functions/fnc_vehicleAction.sqf:137]:
```suggestion
		10,
```

**Review by AdlerSalbei — CHANGES_REQUESTED** [review]
What fussel says!

File: addons/functions/functions/fnc_vehicleAction.sqf (hunk 1 above)
Comment (Fusselwurm) [reply addons/functions/functions/fnc_vehicleAction.sqf:136]:
```suggestion
		"((_target getVariable ['grad_trenches_functions_plowlowered', 0]) == 1) && _this == driver _target", // _target, _this, _originalTarget
```

(purely from what I read in the documentation, I have not tested this)

File: addons/functions/functions/fnc_vehicleAction.sqf [hunk 3]
```diff
@@ -90,7 +90,7 @@ if (hasInterface) then {
 		true,
 		true,
 		"",
-		"((_target getVariable ['grad_trenches_functions_plowlowered', 0]) == 0)", // _target, _this, _originalTarget
+		"((_target getVariable ['grad_trenches_functions_plowlowered', 0]) == 0) && player == driver _target", // _target, _this, _originalTarget
```
Comment (Fusselwurm) [inline addons/functions/functions/fnc_vehicleAction.sqf:93]:
```suggestion
		"((_target getVariable ['grad_trenches_functions_plowlowered', 0]) == 0) && _this == driver _target", // _target, _this, _originalTarget
```

**Pull request merged**